runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -128,6 +146,34 @@ private boolean isInKingdom()
 			&& KINGDOM_REGION.contains(client.getLocalPlayer().getWorldLocation().getRegionID());
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.KINGDOM_GROUP_ID && config.showKingdomValue())
+		{
+			ItemContainer kingdomRewardContainer = client.getItemContainer(InventoryID.KINGDOM_OF_MISCELLANIA);
+			Item[] items = kingdomRewardContainer.getItems();
+			long kingdomPrice = 0;
+
+			for (Item item : items)
+			{
+				long itemStack = (long) itemManager.getItemPrice(item.getId()) * (long) item.getQuantity();
+				kingdomPrice += itemStack;
+			}
+
+			final ChatMessageBuilder message = new ChatMessageBuilder()
+				.append(ChatColorType.HIGHLIGHT)
+				.append("Your kingdom reward is worth around ")
+				.append(StackFormatter.formatNumber(kingdomPrice))
+				.append(" coins.")
+				.append(ChatColorType.NORMAL);
+
+			chatMessageManager.queue(QueuedMessage.builder()
+				.type(ChatMessageType.ITEM_EXAMINE)
+				.runeLiteFormattedMessage(message.build())
+				.build());
+		}
+	}
```

Comment:
Separate this from the next function with a newline.
```suggestion
	}

```